django/contrib/gis/geoip2.py

```diff
@@ -202,18 +202,23 @@ def city(self, query):
         response = self._city.city(enc_query)
         region = response.subdivisions[0] if response.subdivisions else None
         return {
+            "accuracy_radius": response.location.accuracy_radius,
             "city": response.city.name,
             "continent_code": response.continent.code,
             "continent_name": response.continent.name,
             "country_code": response.country.iso_code,
             "country_name": response.country.name,
-            "dma_code": response.location.metro_code,
             "is_in_european_union": response.country.is_in_european_union,
             "latitude": response.location.latitude,
             "longitude": response.location.longitude,
+            "metro_code": response.location.metro_code,
             "postal_code": response.postal.code,
-            "region": region.iso_code if region else None,
+            "region_code": region.iso_code if region else None,
+            "region_name": region.name if region else None,
             "time_zone": response.location.time_zone,
+            # Kept for backward compatibility.
+            "dma_code": response.location.metro_code,
+            "region": region.iso_code if region else None,
```

Comment:
I'm thinking we could deprecate these? We'd need to remove these, wrap the dictionary with something to issue a warning, and add a mapping of replacements, i.e. `{"dma_code": "metro_code", "region": "region_code"}`.

I'm just holding off on working on that if it's preferred to just leave these available as above.

Reply:
If you want to work on it, I'm not against deprecation but I'm not sure it's worth doing. We can also keep them forever or at least for now.